elastic/kibana
# [AO] Improve refreshing alert summary widget on the rule details page

Fixes #147680

## 📝 Summary

This PR improves the refreshing alert summary widget on the rule details page.

Regarding the `execution history` tab, I didn't change the logic since, at the moment, we don't have a way to notify the parent component about refreshing the `execution history` section, so I just focused on the alert summary widget in this PR.

## 🧪 How to test
- Create a rule and go to the rule details page
- Select the alerts tab
- When you refresh the table or change the status filter, you should see a `_alert_summary` API call in the network tab of the browser console to indicate alert summary widget gets updated.

![image](https://user-images.githubusercontent.com/12370520/220165203-c7ee780a-6a19-4b5c-9895-fb1ef6be1839.png)
